Embedded PCK option gives error messages with templates built using LTO.

**Godot version:**
689db6882

MSVC platform=windows bits=32 target=release tools=no **use_lto=yes** 

**OS/device including version:**
Win10 64-bit Editor / 32-bit Release

**Issue description:**
Tried using the export with embedded PCK option with a template that I compiled with LTO using MSVC.

Got this message.
![godot windows tools 64_2020-01-26_09-58-39](https://user-images.githubusercontent.com/13004169/73137052-0c41da80-405d-11ea-8f5b-87f670203448.png)

Traced to:
https://github.com/godotengine/godot/blob/be1bc53d42577c02cbdf32fff5e2a60234e6270a/editor/editor_export.cpp#L1594-L1597

Then debugged it to a loop where it cannot find what it's looking for:
https://github.com/godotengine/godot/blob/be1bc53d42577c02cbdf32fff5e2a60234e6270a/platform/windows/export/export.cpp#L410-L438

Oddly the executable still runs, but I'm very paranoid about it.  It does build fine with external PCK.

@RandomShaper Is it expected not to use LTO when using this feature, or is there something extra that can be done to allow for it?  I imagine LTO can arbitrarily shuffle things around.  Or is this maybe a false error?

**Steps to reproduce:**
Use an export template built with MSVC on Windows using LTO flag.

Additionally had these warnings during the LTO phase when building it, though they don't look relevant.  (Including for completeness)

```
Generating code
C:\Godot\Version 3.2\Source\drivers\gles2\rasterizer_canvas_gles2.cpp(1011) : warning C4750: 'void __thiscall RasterizerCanvasGLES2::_draw_polygon(int const *,int,int,struct Vector2 const *,struct Vector2 const *,struct Color const *,bool,float const *,int const *)': function with _alloca() inlined into a loop
C:\Godot\Version 3.2\Source\drivers\gles2\rasterizer_canvas_gles2.cpp(1032) : warning C4750: 'void __thiscall RasterizerCanvasGLES2::_draw_polygon(int const *,int,int,struct Vector2 const *,struct Vector2 const *,struct Color const *,bool,float const *,int const *)': function with _alloca() inlined into a loop
C:\Godot\Version 3.2\Source\drivers\gles2\rasterizer_canvas_gles2.cpp(1037) : warning C4750: 'void __thiscall RasterizerCanvasGLES2::_draw_generic_indices(unsigned int,int const *,int,int,struct Vector2 const *,struct Vector2 const *,struct Color const *,bool)': function with _alloca() inlined into a loop
C:\Godot\Version 3.2\Source\drivers\gles2\rasterizer_canvas_gles2.cpp(1677) : warning C4750: 'void __thiscall RasterizerCanvasGLES2::_canvas_item_render_commands(struct RasterizerCanvas::Item *,struct RasterizerCanvas::Item *,bool &,struct RasterizerStorageGLES2::Material *)': function with _alloca() inlined into a loop
C:\Godot\Version 3.2\Source\drivers\gles2\rasterizer_canvas_gles2.cpp(1756) : warning C4750: 'void __thiscall RasterizerCanvasGLES2::_canvas_item_render_commands(struct RasterizerCanvas::Item *,struct RasterizerCanvas::Item *,bool &,struct RasterizerStorageGLES2::Material *)': function with _alloca() inlined into a loop
  100%   0 seconds remaining
Finished generating code

```



**Minimal reproduction project:**

I have included the build template in this project for expedience.
SHA256: E6F8423029B2598F5F262B461EB47BB12A28856587A2A68831372ECD1F996448

[Export Embedded PCK Issue.zip](https://github.com/godotengine/godot/files/4113599/Export.Embedded.PCK.Issue.zip)

--- a/platform/windows/godot_windows.cpp
+++ b/platform/windows/godot_windows.cpp
@@ -140,6 +140,11 @@ int widechar_main(int argc, wchar_t **argv) {
 
 	setlocale(LC_CTYPE, "");
 
+#ifndef TOOLS_ENABLED
+	// Workaround to prevent LTCG (MSVC LTO) from removing "pck" section
+	char *dummy_guard = dummy;
+#endif
+
 	char **argv_utf8 = new char *[argc];
 
 	for (int i = 0; i < argc; ++i) {

